docs/references/vod/upload.mdx

```diff
@@ -73,3 +74,52 @@ curl --location --request PUT "${url}" \
 Notice that we only support mp4 files encoded with H264 video and AAC audio for
 now. If you are unsure about these, try sending the file anyway and check any
 returned errors!
+
+# Resumable Uploads
+
+Livepeer Studio supports resumable uploads via [Tus](https://tus.io/).
+This section provides a simple example of how to use `tus-js-client` to upload a video file.
+
+From the previous section, we generated a URL to upload a video file to Livepeer on `POST /api/asset/request-upload`.
+We can use the `tusEndpoint` field of the response to upload the video file and track the progress:
+
+```jsx
+  const file = fs.createReadStream(path);
```

Comment:
I believe we should provide a JavaScript example that works in **the browser**, since that's what most of our developers will be writing this code for. 

As such, we have to replace use of the `fs` package with the corresponding `File` abstractions from the browser. I'm not an expert myself, but I think there's like a `File` class which you can grab from an `<input>` [1] field and then `tus` can take that object directly (and can grab size from it as well)?

[1] I think this part can be omitted, an you add some code just like `const file = document.getElementById("fileInput").getFile()` or something like that, maybe with a short comment saying they should have such element on their page.

WDYT?

Reply:
Sure, I can work on it 😀

Reply:
@Shih-Yu I think it will be easier for @gioelecerati to do it, since I think he had already made a test site during development that uploaded with `tus`. So if there isn't such code already available out there, at least get some pointers with Gio and he should be able to help you!

Reply:
```js
//  Set `input` element 'type="file"' in HTML
input.addEventListener("change", function(e) {

  var file = e.target.files[0];

  var size = file.size;

 const upload = new tus.Upload(file, {
      endpoint: tusEndpoint, // URL from `/request-upload` API response of "tusEndpoint:"
      urlStorage: new (tus as any).FileUrlStorage("/tmp/file"),
      metadata: {
        filename,
        filetype: "video/mp4",
      },
      uploadSize: size,
      onError(error) {
        console.error("Issue with size of file")
      },
      onProgress(bytesUploaded, bytesTotal) {
        const percentage = ((bytesUploaded / bytesTotal) * 100).toFixed(2);
        console.log("Uploaded " + percentage + "%");
      },
      onSuccess() {
        console.log("Upload finished:", upload.url);
      },
    }); 
  })

Reply:
@victorges, @gioelecerati  please code review, this addresses comments made by Victor below as well.

Reply:
Looks good! Some adjustments below. The only bigger change is that I don't think we should suggest putting this logic on a `"change"` event, as that seems like a weird experience in a website. Just wrote the code as if it were a "button click" handler (and fixed some other stuff):
```js
//  Set `input` element 'type="file"' in HTML
const input = document.getElementById("idName");
const file = e.target.files[0];
const upload = new tus.Upload(file, {
    endpoint: tusEndpoint, // URL from `tusEndpoint` field in the `/request-upload` response
    metadata: {
      filename,
      filetype: "video/mp4",
    },
    uploadSize: file.size,
    onError(err) {
      console.error("Error uploading file:", err)
    },
    onProgress(bytesUploaded, bytesTotal) {
      const percentage = ((bytesUploaded / bytesTotal) * 100).toFixed(2);
      console.log("Uploaded " + percentage + "%");
    },
    onSuccess() {
      console.log("Upload finished:", upload.url);
    },
  });
upload.start();
 ```

Reply:
Ok, didn't need to declare `var size` since it could be accessed from within `upload` ...

Reply:
Thanks for this, I changed it to the browser version and made all the other changes!